Expat-Journal/src/App.js

```diff
@@ -6,22 +6,32 @@ import Footer from './components/Footer';
 import HomePage from './components/HomePage';
 import PrivateRoute from './utilities/PrivateRoute';
 import './App.css';
+import UserList from './components/UserList';
+import UserInfo from './components/UserInfo'
 
 function App() {
-  return (
-    <Router>
-      <div className="App">
-      <Header />
-      <Route exact path="/login">
-        <Login />
-      </ Route> 
-      <Footer />
-        {/* <Route exact path="/" component={Login} /> */}
-        {/* <Route exact path="/signup" component = {signup} /> */}
-        <PrivateRoute exact path="/protected" component={HomePage} />
-      </div>
-    </Router>
-  );
-}
+  
+    return (
+      
+      <Router>
+        <div className="App">
+        <Header />
+        <Route exact path="/login">
+          <Login />
+        </ Route> 
+        <Route exact path="/users">
+          <UserList />
+        </ Route> 
+        <Route  path="/users/:id">
+          <UserInfo />
+        </ Route> 
```

Comment:
This should probably be renamed to `/images`, and `ImagesList` or something like that as it has been reworked for images rather then users.